src/components/pill/pill-classic.style.js

```diff
@@ -52,12 +52,14 @@ export default (colorVariant, inFill, isDeletable, size) => {
   return css`
     border-radius: 10px; 
     display: inline-block;
-    font-size: 12px;
+    font-size: 10px;
     font-weight: 700;
-    line-height: 15px;
+    line-height: 10px;
```

Comment:
Apologies in advance if I'm missing something glaringly obvious, has this change been okayed with UX @damianOfKyan? Just asking as I think we are trying to leave `classic` styling "as is" wherever possible to maintain compatibility with legacy uses of the components so I wouldn't expect to see changes to the style properties